core/hydromet_stratified.py

```diff
@@ -0,0 +1,400 @@
+import os
+import io
+import re
+import csv
+import json
+import time
+import urllib
+import shutil
+import logging
+import operator
+import warnings
+import pathlib as pl
+#import papermill as pm
+#import scrapbook as sb
+from zipfile import ZipFile
+from datetime import datetime
+logging.basicConfig(level=logging.ERROR)
+from collections import Counter, OrderedDict
+from IPython.display import display, Markdown
+
+import numpy as np
+import pandas as pd
+from scipy import stats
+from nptyping import Array
+from scipy.optimize import minimize
+from scipy import interpolate, integrate
+from matplotlib import pyplot as plt
+from matplotlib.ticker import FuncFormatter
+from matplotlib.ticker import FormatStrFormatter
+
+import fiona
+import rasterio
+from pyproj import Proj
+import geopandas as gpd
+from rasterio.mask import mask
+from shapely.geometry import mapping
+
+#New additions from the previous hydromet.py file
+from cycler import cycler
+from scipy.integrate import quad
+from scipy import optimize
+
+
+#---------------------------------------------------------------------------#
+
+'''Functions called by EventTable_Stratified.ipynb. 
+'''
+
+#---------------------------------------------------------------------------#
+
+# SCS-CN runoff formula
+def Q_SCS(R,CN,mu):
```

Comment:
Function names and all variables should be lowercase: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names . In python we reserve uppercase for classes and constants. As discussed, please add type hints for each argument, add function descriptions, and remove inline comments.